Makefile

```diff
@@ -163,19 +172,18 @@ ci_debug: $(_TEMPDIR)/ci_debug.tar ## Build and enter container for local develo
 		-e GAC_FILEPATH=$(GAC_FILEPATH) \
 		-e AWS_SHARED_CREDENTIALS_FILE=$(AWS_SHARED_CREDENTIALS_FILE) \
 		-e TEMPDIR=$(_TEMPDIR) \
-		docker-archive:$<
+		$(call imageid,$<) $(if $(DBG_TEST_CMD),$(DBG_TEST_CMD),)
```

Comment:
I don't understand this. Does this magic image have an entrypoint or command defined for when it's invoked without args? If so, did you confirm that the `true` here actually overrides the default? Because sometimes you can override via command-line, and sometimes you need to specify `--entrypoint`. (I could figure this out myself given enough time but my brain hurts from this). Anyhow, CI got skipped for this PR so I can't verify tests.

Reply:
This is intended for interactive-use by humans, and should probably be documented in the README.md.  The entrypoint is simply the default 'bash'.  It's for developing/debugging the dockerfile-based Cirrus-CI tasks, since the image-build happens w/in Cirrus (i.e. it's never pushed anywhere):

(.cirrus.yml)
```yaml
...cut...
imgts_build_task:
    ...cut...
    container:
        dockerfile: "image_builder/Containerfile"
    ...cut...
...cut...
```

> Anyhow, CI got skipped

Careful, CI wasn't entirely skipped.  The builds and new test tasks did run :wink: 

I'll add some quick docs for this.

---

FWIW, the `image_builder_debug` target (also intended for humans) is [already documented](https://github.com/containers/automation_images/blob/main/README.md#debugging--locally-driving-vm-image-production).